test/parallel/test-https-timeout-server-2.js

```diff
@@ -24,15 +24,16 @@
 const common = require('../common');
 if (!common.hasCrypto)
   common.skip('missing crypto');
+const fixtures = require('../common/fixtures');
 
 const assert = require('assert');
 const https = require('https');
 const tls = require('tls');
 const fs = require('fs');
 
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fs.readFileSync(fixtures.path('keys', 'agent1-key.pem')),
```

Comment:
This could be using method `fixtures.readKey` instead of `fs.readFileSync` to get the keys.